index.html

```diff
@@ -0,0 +1,637 @@
+                <p>Thermo Fisher Scientific said the available tests will initially be distributed to approximately 200 labs in the U.S.</p>
+              </div>
+              <div class="d-flex flex-column more-main">
+                <img src="https://d.newsweek.com/en/full/1574110/kamala-harris.jpg?w=95&h=95&f=93a9b70e78d3ebe7201d8591f6b8abbb" alt="">
+                <a href="#">Kamala Harris Would Make the Best Vice Presidential Candidate: Poll</a>
+                <p>Andrew Yang received the second-highest level of support from respondents.</p>
+              </div>
+              <div class="d-flex flex-column more-main">
+                <img id="trump" src="https://d.newsweek.com/en/full/1574277/us-president-donald-trump.jpg?w=95&h=95&f=f910c8f626dd3ded8c120e054716be81" alt="">
+                <a href="#">Report Shows Trump Telling Governors to Get Respirators by 'Yourselves'</a>
+                <p>Governors have been waiting for the federal government to provide them with equipment.</p>
+              </div>
+              <div class="d-flex flex-column more-main">
+                <img src="https://d.newsweek.com/en/full/1574240/oriole-park-mlb.jpg?w=95&h=95&f=3bc810b65fb7dd64bb53139042475b05" alt="">
+                <a href="#">Coronavirus Concerns Close Canada's Borders, Canadians and Americans Exempt</a>
+                <p>"I know this news will spark concern among Canadians traveling abroad," Justin Trudeau said. "I want to assure you that our government will not leave you unsupported."</p>
+              </div>
+              <div class="d-flex flex-column more-main">
+                <img src="https://d.newsweek.com/en/full/1574294/us-army-iraq-war-isis-military.jpg?w=95&h=95&f=bad58ce92cc5f192577221ae0d05b6a7" alt="">
+                <a href="#">U.S. Coalition to Leave Some Iraq Bases After Deadly Week of Vioelnce</a>
+                <p>The U.S.-led coalition told Newsweek it "is re-positioning troops from a few smaller bases" in Iraq, where violence involving U.S. troops, militiamen and militants has recently</p>
+              </div>
+              <div class="d-flex flex-column more-main">
+                <img src="https://d.newsweek.com/en/full/1574186/house.jpg?w=95&h=95&f=05d0a25b64e3d92f1850f73140f17c0e" alt="">
+                <a href="#">40 Republicans Voted Against the 'Families First' Coronavirus Response Bill</a>
+                <p>"We all want to help Americans as we face the spread of coronavirus, but this complete rush job is not the way to do it," said Republican Representative Debbie Lesko.</p>
+
+              </div>
+
+            </div>
+
+          </div>
+          <div class="col-lg-3 col-md-12 opinion d-flex flex-column">
+            <h4 class="heading">OPINION</h4>
+
+            <img src="https://d.newsweek.com/en/full/83499/newt-gingrich.png?w=63&h=63&l=48&t=55&f=d8b4f8dfaad8a5f0f71bffcc6c0d6a77" alt="">
```

Comment:
- [ ] All the titles in this section should be links, have a red hover, and currently are `<p>` tags.

- [ ] The name of the author should be aligned to the left.